lib/tasks/commit_backfill.rake

```diff
@@ -63,7 +56,7 @@ namespace :spree_avatax do
         detaillevel: 'Tax',
       }
 
-      history_result = tax_svc.gettaxhistory(history_request)
+      history_result = SpreeAvatax::Shared.tax_svc.gettaxhistory(history_request)
```

Comment:
these might need to change if we make that tax_svc private per the suggestion above